source/GameData.cpp

```diff
@@ -62,8 +62,6 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 #include <algorithm>
 #include <future>
 #include <iostream>
-#include <map>
-#include <set>
```

Comment:
`<map>` is already included by GameData's own header.
`set<T>` is no longer used by this TU